ledger/complete/mtrie/node/node.go

```diff
@@ -35,10 +35,15 @@ type Node struct {
 	path      ledger.Path     // the storage path (leaf nodes only)
 	payload   *ledger.Payload // the payload this node is storing (leaf nodes only)
 	hashValue []byte          // hash value of node (cached)
-	maxDepth  uint16          // captures the longest path from this node to compacted leafs in the subtree
-	regCount  uint64          // number of registers allocated in the subtree
+	// TODO : to delete
+	maxDepth uint16 // captures the longest path from this node to compacted leafs in the subtree
+	// TODO : to delete
+	regCount uint64 // number of registers allocated in the subtree
```

Comment:
we need them for all nodes to be able to compute it for the root node. I'm hesitant to touch these two while we still exploring ways to optimize cadence storage. I should also mention it would hard to remove these two since they are serialized into checkpointers and requires data migrations and back off models to be able to read them for sporks.